README.md

```diff
@@ -32,40 +30,23 @@ Variables are listed in [variables.json](variables.json).
 **Airflow Connections**
 - `SOLRCLOUD`: An HTTP Connection used to connect to SolrCloud.
 - `AIRFLOW_S3`: An AWS (not S3 with latest Airflow upgrade) Connection used to manage AWS credentials (which we use to interact with our Airflow Data S3 Bucket).
-- `AIRFLOW_CONN_SLACK_WEBHOOK`: Slack failure notifications
-- `FUNCAKE_SLACK_WEBHOOK`: Slack success notifications
+- `slack_api_default`: Used to report DAG run successes and failures to our internal slack channels.
 
 ## Local Development
+Local development relies on the  [Airflow Docker Dev Setup submodule](https://github.com/tulibraries/airflow-docker-dev-setup).
 
-This project uses the UNIX `make` command to build, run, stop, configure, and test DAGS for local development. Steps to get started follow.
-
-### Up and running
-
-Clone https://github.com/tulibraries/ansible_playbook_airflow locally; in a shell at the top level of that repository, run `make up` then git clone/pull your working funcake_dags code to ansible_playbook_airflow/dags/funcake-dags. This shows & runs your development DAG code in the Airflow docker containers, with a webserver at http://localhost:8080.
-
-Open a terminal and execute the following commands:
-
-```
-$ git clone https://github.com/tulibraries/funcake_dags.git
-$ cd funcake_dags
-$ make up
-```
-
-Wait for Airflow and associated servers to build and start up. This may take several minutes.
-
-Open a browser and visit http://localhost:8010 to verify  Airflow server is running.
+This project uses the UNIX `make` command to build, run, stop, configure, and test DAGS for local development. These commands are written to first run the script to change into the submodule directory to use and access the development setup. See the [Makefile](Makefile) for the complete list of commands available.
```

Comment:
"These commands are written to first run the script to change into the submodule directory to use and access the development setup" has three infinitives in a row. Is this better?: "These commands first run a script to change into the submodule directory that is used for the development setup."

Reply:
@ebtoner Thanks for the suggestion, I made that change.